tools/cldr-code/src/main/java/org/unicode/cldr/util/CalculatedCoverageLevels.java

```diff
@@ -13,6 +13,11 @@
 
 
 public class CalculatedCoverageLevels {
+    /**
+     * Assumed level for root. CLDR-16420
+     */
+    final private static Level DEFAULT_ROOT_LEVEL = Level.MODERN;
```

Comment:
This is very strange. Why modern? It's it so it is always included?

Reply:
Yes, so that applications can just filter on the effective level.